### PR TITLE
Add keeper integration test for cancel upkeep

### PR DESCRIPTION
**Pull request by infiloop2**
This is a basic integration test for cancel upkeep.
- Cancel an upkeep
- Observe that its counter does not increase for 1m

This is paired with PR in testing framework repo: https://github.com/smartcontractkit/chainlink-testing-framework/pull/304

### CODE REVIEW

File: integration-tests/go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.26.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-testing-framework v1.1.22-0.20220511162451-b8f31235228e
+	github.com/smartcontractkit/chainlink-testing-framework v1.1.23-0.20220513195554-6b161981da4f
```
Comment (infiloop2) [inline integration-tests/go.mod:11]: Temporarily pointing to commit in https://github.com/smartcontractkit/chainlink-testing-framework/pull/304 . Will update to exact version once it's cut

File: integration-tests/go.mod [hunk 2]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.26.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-testing-framework v1.1.22-0.20220511162451-b8f31235228e
+	github.com/smartcontractkit/chainlink-testing-framework v1.1.23-0.20220516102704-9b6d516359af
```
Comment (infiloop2) [inline integration-tests/go.mod:11]: Temporarily pointing to commit in https://github.com/smartcontractkit/chainlink-testing-framework/pull/304 . Will update to exact version once it's cut

**Comment by infiloop2** [comment]
Note: Smoke tests in CI are failing due to `VRFv2 suite @v2vrf`, unrelated to this change

File: integration-tests/go.mod [hunk 3]
```diff
@@ -4,12 +4,12 @@ go 1.18
...
 require (
 	github.com/ethereum/go-ethereum v1.10.16
-	github.com/onsi/ginkgo/v2 v2.1.3
+	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.26.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-testing-framework v1.1.22-0.20220511162451-b8f31235228e
-	github.com/smartcontractkit/helmenv v1.0.67
+	github.com/smartcontractkit/chainlink-testing-framework v1.2.0
```
Comment (infiloop2) [inline integration-tests/go.mod:11]: I ran `go get github.com/smartcontractkit/chainlink-testing-framework@v1.2.0` which updated all the dependencies

File: integration-tests/smoke/keeper_test.go [hunk 4]
```diff
@@ -34,10 +34,9 @@ var _ = Describe("Keeper suite @keeper", func() {
 			env, err = environment.DeployOrLoadEnvironment(
 				environment.NewChainlinkConfig(
 					environment.ChainlinkReplicas(6, config.ChainlinkVals()),
-					"chainlink-keeper-core-ci",
```
Comment (anirudhwarrier) [inline integration-tests/smoke/keeper_test.go:37]: @infiloop2 any reason why the suffix `-core-ci` to the namespaces were removed across tests? I believe this was done to easily identify ci tests running in the cluster.

Comment (infiloop2) [reply integration-tests/smoke/keeper_test.go:37]: I saw this was changed in testing framework repo: https://github.com/smartcontractkit/chainlink-testing-framework/blob/main/suite/smoke/keeper_test.go#L36 so copied that over with the other changes. Should i add it back?

Comment (anirudhwarrier) [reply integration-tests/smoke/keeper_test.go:37]: I think it should remain as before. This change was introduced [here](https://github.com/smartcontractkit/chainlink/pull/6446/files#diff-0b3c9e31377793dfbe89de3c72462a6146004c64c3fcce0ce127b26cce86a018) , which is after the [changes](https://github.com/smartcontractkit/chainlink-testing-framework/pull/257/files#diff-c60c919044f6f193388142f9ed6754e445b6699a6064303a55630862c1f8ba7c) to the chainlink-testing-framework repo.

cc: @kalverra

Comment (infiloop2) [reply integration-tests/smoke/keeper_test.go:37]: Done

**Comment by infiloop2** [comment]
merging PR with https://github.com/smartcontractkit/chainlink/pull/6646

**Pull request closed without merging**